src/containers/ViewerPanel/index.tsx

```diff
@@ -575,12 +581,11 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
                 />
                 {firstFrameTime !== lastFrameTime && (
                     <div
-                        className={classNames(
-                            styles.bottomControlsContainer,
-                            this.embedDisplaySettings.onlyRenderBottomControls
-                                ? styles.fullWidthBottomControls
-                                : styles.defaultControlSpacing
-                        )}
+                        className={classNames(styles.bottomControlsContainer, {
+                            [styles.fullWidthBottomControls]:
+                                playBackControlsType ===
+                                PlaybackControlsDisplay.BottomOnly,
```

Comment:
if you have a default css property, just put that on the main class, and then override it with a conditional class